sos/plugins/process.py

```diff
@@ -30,5 +30,13 @@ def setup(self):
             "ps auxwwwm",
             "ps alxwww"
         ])
+        self.add_copy_spec([
+            "/proc/*/comm",
+            "/proc/*/limits",
+            "/proc/*/stack",
+            "/proc/*/status",
+            "/proc/*/cpuset",
+            "/proc/*/oom_*"
```

Comment:
I can see uses for it; the main drawback is afaik there's no support to point regular tools like `ps` at a snapshot of this data. I'd be happy for it to go in as long as we have some benchmarking numbers that confirm it's not going to add significantly to the runtime (the size of the data should not be a problem - even on systems with 1000s of running processes).